### DISCUSSION TITLE
Add copy collection entities from one to another.

### DISCUSSION THREAD

**Discussion by muazhari** [milvus-io/milvus#31869] [Ideas & Feature requests]
I need a feature to be able to copy collection entities from one to another. In my case, it will minimize the time needed to search entities by not re-embedding the same data that already exists in another collection. What is left is to copy the entities from another collection to work on the current collection.

If I do it manually using the main backend service, it will be hogging the main backend service because of the amount of data to be copied. Also, I need to have the qualities to be able to implement it ideally if I am forced to create a workaround. I think delegating this function to the Milvus itself is the right choice.

**Comment by yhmo** [comment]
The requirement sounds make sense. Wait @xiaofan-luan to confirm.

**Comment by muazhari** [comment]
There is something extra. You can also implement a search for many collections at once to minimize duplication too.

**Comment by xiaofan-luan** [comment]
@wayblink 
is there a way we can backup and restore?
